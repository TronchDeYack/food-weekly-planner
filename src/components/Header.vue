<template>
  <v-app-bar color="primary" app>
    <v-toolbar-title class="header-title font-weight-black">Food Weekly Planner</v-toolbar-title>
  </v-app-bar>
</template>

<style scoped>
.header-title {
  color: white;
  text-transform: uppercase;
}
</style>
