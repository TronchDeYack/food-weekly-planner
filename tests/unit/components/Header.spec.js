import Header from '@/components/Header.vue';

import { createComponent } from '../../utils';

describe('Header', () => {
  it('Should render.', () => {
    const component = createComponent(Header);
    expect(component).toMatchSnapshot();
  });
});
