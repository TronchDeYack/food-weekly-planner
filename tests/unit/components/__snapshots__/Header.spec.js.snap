// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render. 1`] = `
<nav class="navbar navbar-expand-lg navbar-dark bg-primary"><button type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button> <a href="#" class="navbar-brand">Food Weekly Planner</a>
  <div id="navbarNavAltMarkup" class="collapse navbar-collapse">
    <div class="navbar-nav"><a href="#" class="nav-item nav-link active">Menu</a> <a href="#" class="nav-item nav-link">Recipe</a> <a href="#" class="nav-item nav-link">Shopping</a></div>
  </div>
</nav>
`;
